bot/Dockerfile

```diff
@@ -1,32 +1,32 @@
-FROM python:3.7@sha256:6eaf19442c358afc24834a6b17a3728a45c129de7703d8583392a138ecbdb092
+FROM python:3.7-slim
+
+RUN apt update && \
+  apt-get install --no-install-recommends -y \
+    supervisor \
+    git && \
+  python -m pip install --upgrade pip && \
+  pip install \
+    --no-cache-dir \
+    --root-user-action=ignore \
+    cryptography===37.0.4 \
+    poetry===1.1.15 && \
+  poetry config virtualenvs.in-project true && \
+  groupadd kodiak && \
+  useradd --uid 1000 --gid kodiak kodiak && \
+  mkdir -p /var/app && \
+  chown -R kodiak:kodiak /var/app
 
-RUN set -ex && mkdir -p /var/app
-
-RUN apt-get update && apt-get install -y supervisor
-
-RUN mkdir -p /var/log/supervisor
-
-# use cryptography version for poetry that doesn't require Rust
-RUN python3 -m pip install cryptography===37.0.4
-RUN python3 -m pip install poetry===1.1.13
-
-RUN poetry config virtualenvs.in-project true
+WORKDIR /var/app
 
 COPY supervisord.conf /etc/supervisor/conf.d/supervisord.conf
 
-WORKDIR /var/app
-
-COPY pyproject.toml poetry.lock /var/app/
+COPY --chown=kodiak pyproject.toml poetry.lock /var/app
```

Comment:
This appears to error in CI:

```
Step 5/9 : COPY --chown=kodiak pyproject.toml poetry.lock /var/app
When using COPY with more than one source file, the destination must be a directory and end with a /

Exited with code exit status 1
```

https://app.circleci.com/pipelines/github/chdsbd/kodiak/2818/workflows/f4a433c9-18a4-4a29-9873-7e601d2e5a83/jobs/26706

Reply:
ahh, good point, I just updated it